MvvmCross/Navigation/EventArguments/NavigateEventArgs.cs

```diff
@@ -13,11 +14,13 @@ public NavigateEventArgs()
         {
         }
 
-        public NavigateEventArgs(IMvxViewModel viewModel)
+        public NavigateEventArgs(IMvxViewModel viewModel, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Why are we doing it this way - shouldn't we just make NavigateEventArgs inherit from CancelEventArgs?